crunch/cli.py

```diff
@@ -111,10 +114,34 @@ def setup(
     project_name: str,
     directory: str,
     model_directory_path: str,
+    no_quickstarter: bool,
     quickstarter_name: str,
     show_notebook_quickstarters: bool,
+    notebook: bool,
 ):
-    directory = _format_directory(directory, competition_name, project_name)
+    if notebook:
+        if force:
+            print("notebook `--force` is implicit", file=sys.stderr)
+
+        if no_quickstarter:
+            print("notebook `--no-quickstarter` is implicit", file=sys.stderr)
+
+        if quickstarter_name:
+            print("notebook `--quickstarter-name` is incompatible, ignoring it", file=sys.stderr)
+            quickstarter_name = None
+
+        if show_notebook_quickstarters:
+            print("notebook `--show-notebook-quickstarters` is incompatible, ignoring it", file=sys.stderr)
+            show_notebook_quickstarters = False
+
+        if directory != DIRECTORY_DEFAULT_FORMAT:
+            print("notebook `[directory]` is forced to '.'", file=sys.stderr)
+
+        force = True
```

Comment:
Je l'aurais rapproché du `ìf force` comme pour `show_notebook_quickstarters` `et`quickstarter_name` pour plus de clarté

Reply:
The `if`s are ordered by their variable `@click.option` order.